Vester/Tests/Cluster/HA-Enabled.Vester.ps1

```diff
@@ -5,7 +5,9 @@
 $Title = 'HA State'
 
 # The config entry stating the desired values
-[bool]$Desired = $cfg.cluster.haenable
+if($cfg.cluster.haenable){
```

Comment:
This evaluates `$null` = `$false` = `0`, and the last two are valid values. More on the test changes in general review comments.